doc/config.rst

```diff
@@ -97,7 +129,7 @@ the repository to be the central Hackage server:
 
 The name of the repository is given on the first line, and can be
 anything; packages downloaded from this repository will be cached under
-``~/.cabal/packages/hackage.haskell.org`` (or whatever name you specify;
+``$XDG_CACHE_HOME/cabal/packages/hackage.haskell.org`` (or whatever name you specify;
```

Comment:
Suddenly switch from explicit ~/… to $XDG? For the sake of consistency, I’m leaning to using $XDG stuff everywhere in the docs, so perhaps another euphemisms like $CABAL_DIR instead of explicit paths.

Reply:
I don't have a strong preference.  I use `$XDG_CACHE_HOME` here because the default XDG cache dir is not as well known as `~/.config`.

Reply:
Fair enough. I think it's probably good enough, thank you.